src/lib/utils.js

```diff
@@ -492,3 +495,19 @@ function insideBoundingBoxToBoundingBox(value) {
 
   return insideBoundingBoxStringToBoundingBox(value);
 }
+
+function createDocumentationMessageGenerator(
+  widget,
+  { connector = false } = {}
```

Comment:
```suggestion
  { type } = {}
```

Where type would be an string `"widget" | "connector"`